packages/desktop-client/src/components/mobile/transactions/TransactionListItem.tsx

```diff
@@ -40,7 +40,7 @@ const ROW_HEIGHT = 60;
 type TransactionListItemProps = ComponentPropsWithoutRef<
   typeof ListBoxItem<TransactionEntity>
 > & {
-  isNewTransaction: (transaction: TransactionEntity['id']) => boolean;
+  isNewTransaction?: (transaction: TransactionEntity['id']) => boolean;
```

Comment:
I don't see this prop being used anywhere, I am not sure if I should delete it or leave that as an optional